events/dbpersist_test.go

```diff
@@ -16,15 +16,10 @@ import (
 	pds "github.com/bluesky-social/indigo/pds/data"
 	"github.com/bluesky-social/indigo/repomgr"
 	"github.com/bluesky-social/indigo/util"
-	logging "github.com/ipfs/go-log/v2"
 	"gorm.io/driver/sqlite"
 	"gorm.io/gorm"
 )
 
-func init() {
-	logging.SetAllLoggers(logging.LevelDebug)
```

Comment:
I think there is a way to call `slog.SetDefaultLogger`, just to know that, but don't need to do it here and now

Reply:
a little extra util code should set the logger at something that writes to `testing.T.Log()`